accounts_from_snapshot/examples/import_archive.rs

```diff
@@ -0,0 +1,43 @@
+use std::path::PathBuf;
+use std::str::FromStr;
+use std::time::Instant;
+use log::{info, trace};
+use lite_accounts_from_snapshot::import::import_archive;
```

Comment:
(nit) imports sorting and only used ones:

```suggestion
use lite_accounts_from_snapshot::import::import_archive;
use log::{debug, info, trace};
use std::time::Instant;
```